README.md

```diff
@@ -9,24 +9,32 @@ Started](https://dvc.org/doc/get-started) and other sections of the DVC docs.
 Please make sure you have these available on the environment where these scripts
 will run:
 
-- Git
-- Python (with `pip`)
+- [Git](https://git-scm.com/)
+- [Python](https://www.python.org/) 3 (with `python3` and [pip](https://pypi.org/project/pip/) commands)
```

Comment:
it's actually should be possible to run it with Python 2, no? as least get started itself is compatible with Python 2

Reply:
Probably. The [script](https://github.com/iterative/example-repos-dev/pull/6/files#diff-95ce536a3300fc445a5e3964c5e6e8bcR32) already has `virtualenv -p python3 .env` though so I just documented that as a requirement. Should I change it? Or maybe just check that it works on Python 2 and not specify a version in the script and README?

Reply:
no, you are right. let's keep it. It's just I'm running it manually from time to time with Python 2 to make sure that everything works as expected. But when we generate it, it's probably better to use the same version every time. I'll think if we need an option to run it w Python 2.

Reply:
Lmk